Выполнила первый проект "С чистого листа"

ESKuchina opened the pull request:
Выполнила первый проект "С чистого листа" и отправляю на первое ревью в команду Яндекс.Практикум

ESKuchina committed:
Выполнила первый проект "С чистого листа"

Выполнила первый проект "С чистого листа" и отправляю на первое ревью в команду Яндекс.Практикум

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+          alt="Логотип Вёрстка"
+          class="header__logo-image"
+      /></a>
+      <nav class="header__menu">
```
**Отлично:**

Отлично, что использовали тег `nav` для списка ссылок навигации

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+      /></a>
+      <nav class="header__menu">
+        <ul class="header__menu-list">
+          <li class="header__menu-item header__menu-item_active">Главная</li>
```
**Надо исправить:**

`Главная` так же является ссылкой, её тоже следует обернуть в тег ссылки (`a`)

ESKuchina replied:
Обернула "Главная" в тег ссылки (а), но не совсем понимаю, зачем это делать? В рамках работы я верстаю экран главной страницы, на которой по факту мы и находимся, и получается, что "Главная" в меню уже и не ссылка, - я же не могу перейти на страницу, на которой итак нахожусь. А, раз не могу перейти, то зачем оборачивать в тег ссылки (а)? Или я что-то не так понимаю?

Mililiya replied:
Здравствуйте! В будущем проект может разрастись, соответственно может появиться возможность перехода между страницами и т.д.  Чтобы на данном этапе у ссылки `Главная` не было никакого взаимодействия с ней, стоит убрать атрибут `href="#0"` и тогда она станет недоступна для клика

ESKuchina replied:
Спасибо! Теперь всё поняла)))

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        </ul>
+      </nav>
+    </header>
+    <main>
```
**Отлично:**

Отлично, что для основного содержимого страницы использовали тег `main`

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+    </header>
+    <main>
+      <section class="cover content-section content-section_theme_dark">
+        <h1 class="cover__title">
```
**Отлично:**

Верно подобрали уровень заголовка, отлично что `h1` только один в проекте

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        </h1>
+      </section>
+      <section class="intro content-section content-section_theme_dark">
+        <header class="intro__heading">
```
**Надо исправить:**

Сущность служит для группировки контента внутри, тут уместно использовать `div`, вместо `header`

ESKuchina replied:
Поняла :) Поправила

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+          <h2 class="intro__title">
+            8 способов борьбы со страхом чистого листа
+          </h2>
+          <time datetime="2022-05-06" class="intro__date">05.06.2022</time>
```
**Надо исправить:**

Контентное содержимое следует установить по макету - `05.06.2022 г.`

ESKuchina replied:
Внимательность явно не мой конек :'( Поправила

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+          состояние потока. Причин, почему первые усилия даются так сложно,
+          может быть очень много. Вот основные:
+        </p>
+        <ul class="content-section__list">
```
**Отлично:**

Отличное использование тега списка `ul > li`

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+        <ul class="cards-list">
+          <li>
+            <article class="card card_theme_dark">
+              <section class="card__author">
```
**Надо исправить:**

Сущность не является отдельной секцией, тут стоит использовать `div` для группировки контента

Проконтролируйте это тут и далее в блоке

ESKuchina replied:
Поняла :) Поправила

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+                />
+                <span class="card__author-name">Алексей Поляринов</span>
+              </section>
+              <section class="card__content">
```
**Надо исправить:**

Сущность не является отдельной секцией, тут стоит использовать `div` для группировки контента

Проконтролируйте это тут и далее в блоке

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+                  alt="Фотография автора Алексея Поляринова"
+                  class="card__author-photo"
+                />
+                <span class="card__author-name">Алексей Поляринов</span>
```
**Можно лучше:**

Для текстового элемента так же можно воспользоваться тегом `p`

Mililiya commented on index.html:
```diff
@@ -0,0 +1,207 @@
+      </section>
+    </main>
+    <footer class="footer content-section content-section_theme_dark">
+      <section class="footer__copy">&copy; VЁRSTKA</section>
```
**Надо исправить:**

Для подписи в `footer` следует использовать тег `p`

ESKuchina replied:
Поняла :) Поправила

Mililiya commented on styles/style.css:
```diff
@@ -0,0 +1,248 @@
+}
+
+.header {
+  height: 132px;
```
**Надо исправить:**

Необходимо удалить высоту у всех контентных и текстовых блоков. Для правильного расположения элементов необходимо применять технологию выравнивания (высоту строки, отступы) и построения сетки (флекс/грид)

ESKuchina replied:
Поняла :) Поправила

Mililiya commented on styles/style.css:
```diff
@@ -0,0 +1,248 @@
+
+.content-section__list-item {
+  line-height: 135%;
+  margin-left: 20px;
```
**Можно лучше:**

Более точным значением будет 16px

ESKuchina replied:
Спасибо ;) Поправила

Mililiya reviewed (commented):
Здравствуйте. (Нужно развернуть общий комментарий ↓)
​
Работа была проделана большая, но есть моменты которые хотелось быть отметить:
​
*Отлично:*
​
* Отлично структурированный `html` код, страница валидна
* Отлично, что использовали тег `nav` для списка ссылок навигации
* Отлично, что для основного содержимого страницы использовали тег `main`
* Верно подобрали уровень заголовка, отлично что `h1` только один в проекте
* Комментарии в коде, отмеченные как `Отлично`
​
*Надо исправить:*
​
* Комментарии в коде, отмеченные как `Надо исправить`
* https://disk.yandex.ru/i/BgNx0Sf_f_Jzyg - сейчас в проекте отсутствует логотип
* Все изображения из корня проекта следует переместить в соответствующую папку `images`
* `index.html` должен быть расположен в корне каталога, лишний файл `index.html` из папки `styles` следует удалить. 
* Файл стилей `style.css` должен храниться в соответствующей папке `styles`, а из корня каталога файл следует удалить, так же с файлом шрифтов.
 
*Можно лучше:*
​
* Комментарии в коде, отмеченные как `Можно лучше`
​
​
Обратите внимание, что работа будет принята только после исправления всех комментариев `Надо исправить`, комментарии `Можно лучше` не обязательны к исправлению.
​
Удачной работы, до встречи на следующем ревью.

ESKuchina committed:
Add files via upload

ESKuchina committed:
Delete bunin.png

Удалила картинку из корня, - почему-то файл задублировался при переносе в ветку и был и в корне, и в папке images

ESKuchina committed:
Delete cover.svg

Удалила картинку из корня, - почему-то файл задублировался и при переносе файлов с компьютера попал и в корневую попку, и в images

ESKuchina committed:
Delete ebgaramondmedium.ttf

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete ebgaramondmedium.woff

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete ebgaramondmedium.woff2

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete ebgaramondregular.ttf

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete ebgaramondregular.woff

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete ebgaramondregular.woff2

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete fonts.css

Удалила дубль fonts.css из корневой папки. Файл лежит в папке fonts

ESKuchina committed:
Delete interbold.ttf

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete interbold.woff

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete interbold.woff2

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete interregular.ttf

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete interregular.woff

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete interregular.woff2

Удалила файл со шрифтов из корневой папки. Все файлы со шрифтами расположены в папке fonts

ESKuchina committed:
Delete logo.svg

Удалила картинку из корня, - почему-то файл задублировался и при переносе файлов с компьютера попал и в корневую попку, и в images

ESKuchina committed:
Delete polyarinov.png

Удалила картинку из корня, - почему-то файл задублировался и при переносе файлов с компьютера попал и в корневую попку, и в images

ESKuchina committed:
Delete style.css

Удалила дубль файла style.css из корневой папки. Файл лежит в папке styles

ESKuchina committed:
Delete styles/index.html

Удалила дубль файла index.html из папки styles. Файл index.html лежит в корневой папке

ESKuchina committed:
Внесла правки, после первого ревью

Внесла правки, в соответствии с комментариями/замечаниями от ревьюера команды Яндекс.Практикум

ESKuchina commented:
Не уверена, что всё правильно залила в GitHub, - не работала с ним до курса :( Вроде файлы все обновила в ветке review они отображаются корректно. Но не совсем понимаю, как изменения отображаются тут на вкладке Pull requests. Надеюсь, что всё корректно.

Mililiya reviewed (commented):
Поздравляю! Ваша работа принята.

Вы отлично потрудились.

Удачного дальнейшего обучения.

ESKuchina commented:
Ууууууураааааааа!!!!! Спасибо!)
